### PR TITLE
Issue #73: Add support for mailing list

### PR DESCRIPTION
**Pull request by Vladlis**

### CODE REVIEW

File: releasenotes-builder/config/import-control.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <allow pkg="java.net"/>
     <allow pkg="java.nio"/>
     <allow pkg="java.util"/>
+    <allow pkg="javax.mail"/>
```
Comment (romani) [inline releasenotes-builder/config/import-control.xml]: should be allowed only in "publishers" subpackge.

File: releasenotes-builder/README.md [hunk 2]
```diff
@@ -75,6 +76,14 @@ release notes. For example, ```753bc06``` or just tag ```checkstyle-6.12.1``` .
 
 **twitterProperties** - (optional) properties for connection to Twitter.
 
+**publishTwit** - (optional) publish to mailing list from ```mailing_list.txt```.
```
Comment (romani) [inline releasenotes-builder/README.md]: Twit ?

File: releasenotes-builder/README.md [hunk 3]
```diff
@@ -75,6 +76,14 @@ release notes. For example, ```753bc06``` or just tag ```checkstyle-6.12.1``` .
...
+
+**mlistUsername** - (optional) username key for mailing list.
+
+**mlistPassword** - (optional) password key for mailing list.
+
+**mlistProperties** - (optional) properties for publication to mailing list.
```
Comment (romani) [inline releasenotes-builder/README.md]: is it a path to property file ? if yes please clearly state that in description.

File: releasenotes-builder/README.md (hunk 2 above)
Comment (Vladlis) [inline releasenotes-builder/README.md]: fixed

File: releasenotes-builder/README.md (hunk 3 above)
Comment (Vladlis) [inline releasenotes-builder/README.md]: done

**Pull request merged**